src/mds/Server.cc

```diff
@@ -7273,6 +7273,9 @@ void Server::handle_client_getvxattr(const MDRequestRef& mdr)
       // since we only handle ceph vxattrs here
       r = -ENODATA; // no such attribute
     }
+  } else if (xattr_name == "ceph.dir.subvolume"sv) {
+    const auto srnode = cur->get_projected_srnode();
+    *css << ((!srnode || !srnode->is_subvolume()) ? "0" : "1");
```

Comment:
It would be nice if we can add tests for exercising this vxattr so that we can catch such bugs in future.